tests/simple_isthmus_transition.rs

```diff
@@ -0,0 +1,15 @@
+use common::RollupBoostTestHarnessBuilder;
+
+mod common;
```

Comment:
I had to create separated files for each one because there are some global setters in Rollup-boost that would crash if they were set twice (random, tracing, metrics). I will leave for a follow up PR to simplify this if we want to have all these simple tests in a single file.